src/features/defi/components/Overview/UnderlyingAssetsMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import { Box, Flex, Popover, PopoverBody, PopoverContent, PopoverTrigger } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+
+import { PairIcons } from '../PairIcons/PairIcons'
+import type { AssetWithBalance } from './Overview'
+import { UnderlyingAssetsTags } from './UnderlyingAssetsTags'
+
+type UnderlyingAssetsMenuProps = {
+  underlyingAsset: AssetWithBalance
+  underlyingAssets: AssetWithBalance[]
+}
+export const UnderlyingAssetsMenu = ({
+  underlyingAsset,
+  underlyingAssets,
+}: UnderlyingAssetsMenuProps) => (
+  <Box>
+    <Popover matchWidth>
+      <PopoverTrigger>
+        <Box cursor='pointer'>
+          <UnderlyingAssetsTags underlyingAssets={[underlyingAsset]} showPercentage={false}>
+            <ChevronDownIcon />
+          </UnderlyingAssetsTags>
+        </Box>
+      </PopoverTrigger>
+      <PopoverContent minWidth='200px'>
+        <PopoverBody>
+          {underlyingAssets.map(asset => (
```

Comment:
We can't reuse `<UnderlyingAssetTag` because we need to wrap these in `<Flex />` - maybe a HOC will do the trick?